test/request.js

```diff
@@ -1,6 +1,6 @@
 import stream from 'stream';
 import http from 'http';
-import polyfill from 'abortcontroller-polyfill/dist/abortcontroller.js';
+import AbortController from 'abort-controller';
```

Comment:
If we are going to swap out `abortcontroller-polyfill` for `abort-controller`, the same should be done for `test/main.js`

Reply:
We are checking both, `abortcontroller-polyfill` and `abort-controller` at `test/main.js`. Here we need just one, and `abort-controller` is easier to import from ESM, so, I swap it here.